components/form-file-button/index.js

```diff
@@ -0,0 +1,51 @@
+/**
```

Comment:
I try to group all the "form" related components by calling them `Form*` but I don't mind changing.